Documentation/check-crd-compat-table.sh

```diff
@@ -136,7 +136,8 @@ if [[ "$#" -ne 1 ]]; then
   exit 1
 fi
 
-release_version="${1}"
+release_ersion="$(echo $1 | sed 's/^v//g')"
+release_version="v$release_ersion"
```

Comment:
```suggestion
release_version="${1#v}"
```
should do the trick as well? Or am I missing something with this `g` flag at the end, I don't expect the `v` to be present multiple times?

Reply:
Could potentially do the trick. Agree the `g` is not necessary, since the `^v` will only match on v at the start of the stream anyway. I'm not at all familiar with that bashism, and I suspect that the average contributor will have more familiarity with sed rather than this terse approach.

We've used the sed approach multiple times throughout the tree, I'd suggest if we want to switch over to this shell function instead we can change them all at once. I'll skip this here for now.

Reply:
I dropped the 'g' for what it's worth.